app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/AttachPhotoUtils.java

```diff
@@ -0,0 +1,136 @@
+package de.westnordost.streetcomplete.data.osmnotes;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Matrix;
+import android.support.media.ExifInterface;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import de.westnordost.streetcomplete.util.ImageUploader;
+
+public class AttachPhotoUtils
+{
+	public static String uploadAndGetAttachedPhotosText(ImageUploader imageUploader, List<String> imagePaths)
+	{
+		if(imagePaths != null)
+		{
+			List<String> urls = imageUploader.upload(imagePaths);
+			if (urls != null)
+			{
+				StringBuilder sb = new StringBuilder("\n\nAttached photo(s):");
+				for(String link : urls)
+				{
+					sb.append("\n");
+					sb.append(link);
+				}
+				return sb.toString();
+			}
+		}
+		return "";
+	}
+
+	public static void deleteImages(List<String> imagePaths)
+	{
+		if(imagePaths != null)
+		{
+			for (String path : imagePaths)
+			{
+				File file = new File(path);
+				if (file.exists())
+				{
+					file.delete();
+				}
+			}
+		}
+	}
+
+	public static Bitmap resize(String imagePath, int maxWidth)
```

Comment:
should probably rename to createScaledBitmap, analogous to the Bitmap static method.